launchable/utils/authentication.py

```diff
@@ -15,6 +17,16 @@ def get_org_workspace():
     return os.getenv(ORGANIZATION_KEY), os.getenv(WORKSPACE_KEY)
 
 
+def ensure_org_workspace() -> Tuple[str, str]:
```

Comment:
add `ensure_org_workspace` to assure the parameters are fine, else raise exception